params/Zhang's Rabbit.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 3
+    
+# e.g. "anas_krystallinus"    
+species.name <- " Zhang's Rabbit "
+    
+# e.g. "red"    
+color <- " blue"
```

Comment:
I think there is a tpyo here and an extra space...
```suggestion
color <- "blue"
```